operators/test/e2e/framework/kibana/http_client.go

```diff
@@ -0,0 +1,53 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package kibana
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path"
+
+	"github.com/elastic/cloud-on-k8s/operators/pkg/controller/kibana/name"
+	"github.com/elastic/cloud-on-k8s/operators/test/e2e/framework"
+)
+
+// DoKibanaReq executes an HTTP request against a Kibana instance.
+func DoKibanaReq(b Builder, k *framework.K8sClient, method string, uri string, body []byte) ([]byte, error) {
```

Comment:
I wonder if this should go into the `KibanaClient` which exists in `master` but maybe not in this PR

Reply:
I think so.
I'm currently merging master to fix conflicts.
Then I'm looking to change that.

Reply:
There is no `KibanaClient` struct, just a constructor that returns an `*http.Client`.
So I was just careful to put `DoKibanaReq` in the same file (`operators/test/e2e/test/kibana/http_client.go`).

Reply:
Right!